ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/v2/ShopperInsightsClientV2.kt

```diff
@@ -123,4 +125,26 @@ class ShopperInsightsClientV2 internal constructor(
     fun isVenmoAppInstalled(context: Context): Boolean {
         return deviceInspector.isVenmoInstalled(context)
     }
+    /**
+     * Updates the customer session with the given session ID.
```

Comment:
Could we update this kdoc to include param annotations similar to the doc above for `createCustomerSession()`?

```kotlin

    /**
     * Creates a new customer session.
     *
     * @param customerSessionRequest: a [CustomerSessionRequest] object containing the request parameters
     * @param customerSessionCallback: a callback that returns the result of the customer session creation
     *
     * Note: **This feature is in beta. It's public API may change in future releases.**
     */
```

It may also be worth adding the note about the feature being in beta.